packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/selection/SelectionControllerIntegrationTests.java

```diff
@@ -118,6 +121,9 @@ subscription selectionDialogTreeEvent($input: SelectionDialogTreeEventInput!) {
     @Autowired
     private RepresentationIdBuilder representationIdBuilder;
 
+    @Autowired
+    private ObjectMapper objectMapper;
```

Comment:
You don't need that, just write the proper value, it's not that hard

Reply:
I have probably seen it written like that in another test :-)